arbitrator/src/arbitrator.cpp

```diff
@@ -107,6 +107,8 @@ namespace arbitrator
             initialized_ = true;
             // TODO: load plan duration from parameters file
         }
+
+        ros::Duration(0.1).sleep();
```

Comment:
Do we need this in planning_state as well?

Reply:
We shouldn't need it I think. Once planning state wraps up it transitions to waiting state where we do have this sleep while it waits out the 1s until the next plan generation.